src/components/Icon/Icon.tsx

```diff
@@ -140,11 +138,12 @@ export const Icon = (props: IconProps) => {
     style,
     width: size,
     xmlns: 'http://www.w3.org/2000/svg',
-    viewBox,
+    viewBox: name ? icons[name].viewBox : viewBox,
   };
+
   // allow passing custom SVGs to render, otherwise
   // load from the spritemap of EDS icons
-  const computedSvg = children || <use xlinkHref={`${icons}#${name}`} />;
+  const computedSvg = name ? icons[name].content : children;
```

Comment:
Note the logic change here to match `viewBox` above.
- Before - `children` took precedence over `name` (if you provide both, you get `children`)
- Now - `name` takes precedence over `children` (if you provide  both, you get the named icon)

Can revert this if it seems risky or undesired.